<!--?xml version="1.0" encoding="UTF-8"?-->
<testsuite time="[time]" timestamp="[timestamp]" tests="3" skipped="0" failures="3">
  <testcase file="error_in_js_classic.spec.html" name="chromium" time="[time]" timestamp="[timestamp]" tests="1" errors="1">
    <properties>
      <property name="runtimeName" value="chromium"></property>
      <property name="runtimeVersion" value="130.0.6723.31"></property>
    </properties>
  </testcase>
  <error type="Error">1 | throw new Error("here");
          ^
Error: here
  at base/client/error_in_js_module.spec.js:1:7</error>
  <testcase file="error_in_js_classic.spec.html" name="firefox" time="[time]" timestamp="[timestamp]" tests="1" errors="1">
    <properties>
      <property name="runtimeName" value="firefox"></property>
      <property name="runtimeVersion" value="131.0"></property>
    </properties>
  </testcase>
  <error type="Error">1 | throw new Error("here");
          ^
Error: here
@base/client/error_in_js_module.spec.js:1:7</error>
  <testcase file="error_in_js_classic.spec.html" name="webkit" time="[time]" timestamp="[timestamp]" tests="1" errors="1">
    <properties>
      <property name="runtimeName" value="webkit"></property>
      <property name="runtimeVersion" value="18.0"></property>
    </properties>
  </testcase>
  <error type="Error">1 | throw new Error("here");
                   ^
Error: here
module code@base/client/error_in_js_module.spec.js:1:16</error>
</testsuite>